src/host/input.cpp

```diff
@@ -306,7 +306,7 @@ void ProcessCtrlEvents()
         break;
 
     case CONSOLE_CTRL_C_FLAG:
-        EventType = CTRL_C_EVENT;

```

Comment:
what's this?

Reply:
Huh, I was certain I reverted that. It was for testing.